packages/connector-types/src/aliyun.ts

```diff
@@ -0,0 +1,82 @@
+import { createHmac } from 'crypto';
```

Comment:
is this file required over aliyun sms/email and social? will it be better to specify what would import this

Reply:
Should reorganize these constants and methods in aliyun.ts.
They are initially added for aliyun-dm and aliyun-sms connectors and are not general enough. We may simply put code in aliyun.ts in connector-aliyun-sms and connector-aliyun-dm, or wrap them as a new package (such as @logto/aliyun-utils).
At this stage, I personally prefer the first solution as we do not have many aliyun-based services.

Reply:
Agree with first solution.

Is not too late to refactor when we have 3rd aliyun's connector.